testapps/ngx/README.md

```diff
@@ -4,11 +4,14 @@
 
 To learn more about Tolgee, visit [https://tolgee.io](https://tolgee.io).
 
+## The Docs
+[Read more about using Tolgee with Angular in the docs.](https://tolgee.io/js-sdk/integrations/angular/installation)
+
 ## To run the app in dev mode
 
 To install dependencies, run:
 
-    npm install
+    pnpm install
```

Comment:
I would keep the npm here, since the app is released separately as example app, and it's easier to use npm there, since it's not in any monorepo.